trace/tests/unit/test__gax.py

```diff
@@ -0,0 +1,470 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import mock
+
+from google.cloud._testing import _GAXBaseAPI
+
+
+class _Base(object):
+    project = 'PROJECT'
+
+    def _make_one(self, *args, **kw):
+        return self._get_target_class()(*args, **kw)
+
+
+class Test__TraceAPI(_Base, unittest.TestCase):
```

Comment:
Hah, no - my master plan is just pytest everywhere. @dhermes is on the fence, @lukesneeringer *might* be part of the secret pytest cabal. I quite certain that @tseaver is resolutely in the unittest camp. :)